code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,796 @@
+
+/*
+Warping extracts crossbreed
+put up a rune with passive effects, lots of those runes are fluff or act as a passive buff, others are just griefind tools.
+
+Note : Entered() doesn't seem to work for whatever reason so I had to use Initialize and Signals together, convoluted but it works.
+*/
+
+/obj/item/slimecross/warping
+	name = "warping extract"
+	desc = "It just won't stay in place."
+	icon_state = "warping"
+	effect = "warping"
+	colour = "grey" ///default color of the crossbreed
+	var/obj/effect/warped_rune/runepath ///what runes will be drawn depending on the crossbreed color
+	var/warp_charge = 1 /// the number of "charge" a bluespace crossbreed start with
+	var/max_charge = 1 ///max number of charge, might be different depending on the crossbreed (all crossbreed have 1 max charge for now)
+	var/storing_time = 15 ///time it takes to store the rune back into the crossbreed
+	var/drawing_time = 15 ///time it takes to draw the rune
+
+/obj/effect/warped_rune
+	name = "warped rune"
+	desc = "An unstable rune born of the depth of bluespace"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "greyspace_rune"
+	var/runepath = /obj/effect/warped_rune  ///serves as a way for some specific runes to identify themselves in the turf.TODO: you can just use src dumbass
+	var/storing_time = 15 ///should always be the same as the storing_time of the slimecross,is only used for bluespace crystal erasing as of now
+	anchored = TRUE
+	layer = MID_TURF_LAYER
+	resistance_flags = FIRE_PROOF  //It's not acidproof like cult runes to give another way to delete them
+	var/turf/T //used on a LOT of runes, so we might as well put it here
+	var/cooldown = 0 //cooldown for the rune process() only applies to certain rune
+	var/max_cooldown = 100
+
+///runes can also be deleted by bluespace crystals much faster than with soap/mops
+/obj/effect/warped_rune/attackby(obj/item/stack/BC, mob/user)
+	if(!istype(BC,/obj/item/stack/sheet/bluespace_crystal) && !istype(BC,/obj/item/stack/ore/bluespace_crystal))
+		return
+	if(do_after(user, storing_time,target = src)) //the time it takes to nullify it depends on the rune too
+		to_chat(user, "<span class='notice'>You nullify the effects of the rune with the bluespace crystal!</span>")
+		qdel(src)
+		BC.amount--
+		playsound(src, 'sound/effects/phasein.ogg', 20, TRUE)
+		if(BC.amount <= 0)
+			qdel(BC)
+///destroyable by xenomorphs			
+/obj/effect/warped_rune/acid_act() 
+	visible_message("<span class='warning'>The rune is being disolved!</span>")
```

Comment:
*dissolved

also, if the rune is dissolved instantly by acid, why are you giving a message that it's "being" dissolved (implying that it's still dissolving)?

also, you might wanna use "[src]" instead of "the rune"